src/python/gudhi/rips_complex.pyx

```diff
@@ -41,31 +41,30 @@ cdef class RipsComplex:
     cdef Rips_complex_interface thisref
 
     # Fake constructor that does nothing but documenting the constructor
-    def __init__(self, points=None, distance_matrix=None,
+    def __init__(self, *, points=None, distance_matrix=None,
                  max_edge_length=float('inf'), sparse=None):
         """RipsComplex constructor.
 
-        :param max_edge_length: Rips value.
-        :type max_edge_length: float
-
         :param points: A list of points in d-Dimension.
-        :type points: list of list of float
+        :type points: List[List[float]]
```

Comment:
At some point, we should be consistent with WeightedRipsComplex (`Sequence[Sequence[float]]`) and DTMRipsComplex (`numpy.ndarray`)